packages/core/src/extensions/renderer-api/k8s-api.ts

```diff
@@ -110,7 +112,8 @@ export type { SecretStore as SecretsStore } from "../../renderer/components/+con
 export type { ReplicaSetStore } from "../../renderer/components/+workloads-replicasets/store";
 export type { ResourceQuotaStore as ResourceQuotasStore } from "../../renderer/components/+config-resource-quotas/store";
 export type { LimitRangeStore as LimitRangesStore } from "../../renderer/components/+config-limit-ranges/store";
-export type { HorizontalPodAutoscalerStore as HPAStore } from "../../renderer/components/+config-autoscalers/store";
+export type { HorizontalPodAutoscalerStore as HPAStore } from "../../renderer/components/+config-horizontal-pod-autoscalers/store";
+export type { VerticalPodAutoscalerStore as VPAStore } from "../../renderer/components/+config-vertical-pod-autoscalers/store";
```

Comment:
I don't think we need to add to the API with a renamed export.

Reply:
There was a reason it was done for HPA?

Reply:
Backwards compatibility, the name got changed internally by we don't want to break the extension API.

Reply:
So we'll have `HPAStore` and `VerticalPodAutoscalerStore`? Seems unbalanced Or is `HorizontalPodAutoscalerStore` also exported already?

Reply:
No we should export `HorizontalPodAutoscalerStore` and mark `HPAStore` as depcrecated. Basically all the other stores use the whole name so this is more of the outlier.

Reply:
like this?
```
/**
 * @deprecated
 */
export type { HorizontalPodAutoscalerStore as HPAStore } from "../../renderer/components/+config-horizontal-pod-autoscalers/store";
export type { HorizontalPodAutoscalerStore } from "../../renderer/components/+config-horizontal-pod-autoscalers/store";
export type { VerticalPodAutoscalerStore } from "../../renderer/components/+config-vertical-pod-autoscalers/store";
```

Reply:
Close:

```ts
export type { 
  /**
   * @deprecated
   */
  HorizontalPodAutoscalerStore as HPAStore,
} from "../../renderer/components/+config-horizontal-pod-autoscalers/store";
export type { HorizontalPodAutoscalerStore } from "../../renderer/components/+config-horizontal-pod-autoscalers/store";
export type { VerticalPodAutoscalerStore } from "../../renderer/components/+config-vertical-pod-autoscalers/store";
```